front/lib/models/assistant/agent.ts

```diff
@@ -88,6 +92,7 @@ export class AgentConfiguration extends Model<
   declare version: number;
 
   declare status: AgentStatus;
```

Comment:
What is Exclude ?

Reply:
New type = provided type - provided values